src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -58,6 +57,25 @@ export class LWCEslintStrategy implements EslintStrategy {
 		return paths;
 	}
 
+	filterDisallowedRules(rulesByName: Map<string, ESRule>): Map<string, ESRule> {
+		const filteredRules: Map<string,ESRule> = new Map();
+		for (const [name, rule] of rulesByName.entries()) {
+			// Keep all rules except the deprecated ones.
+			if (!rule.meta.deprecated) {
```

Comment:
Can we create another EslintStrategyHelper method to handle this - Javascript and Typescript's implementation looks similar to me.

Reply:
Sure. I'll do that shortly.